sdk/servicebus/azservicebus/samples/processor/processor_sample.go

```diff
@@ -0,0 +1,106 @@
+package main
```

Comment:
There is a samples directory inside of `sdk`, do we want to put samples there or are we moving to having samples in each module?

Reply:
That's a great question! 

@jhendrixMSFT , @RickWinter - should I move these into `sdk/samples`? 

We had both kinds of samples in JS - the package samples would demo the package, and the repo level samples were more general and covered things like browser bundling, etc...